bin/metrics-elb.rb

```diff
@@ -117,8 +117,17 @@ def print_statistics(load_balancer_name, statistics)
     static_value = {}
     statistics.each do |key, static|
       r = cloud_watch_metric(key, static, load_balancer_name)
-      static_value['loadbalancer.' + load_balancer_name + '.' + key + '.' + static] = static
-      result['loadbalancer.' + load_balancer_name + '.' + key + '.' + static] = r[:datapoints][0] unless r[:datapoints][0].nil?
+      keys =
+        if config[:scheme].nil?
```

Comment:
What scenario would this be `nil` in other than explicitly setting setting it to `nil`? It previously had a default of `''` (empty string) which is not the same as nil:
 ```ruby
$ irb
irb(main):001:0> a = ''
=> ""
irb(main):002:0> a.nil?
=> false
```
I can't see a scenario where it is even possible, it accepts a string value so even if you specified nil I would imagine it would return the string of `'nil'`

Also the if should go on the same line as the assignment.

Reply:
Good catch, the test should be on the empty string instead of ``nil``, I copied/pasted the wrong code from another plugin apparently...

Reply:
I also moved the ``if`` on the same line as the assignment, although it's not very consistent with the other checks :)